include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -5251,6 +5251,22 @@ struct OrtApi {
    * \since Version 1.22.
    */
   const OrtEpApi*(ORT_API_CALL* GetEpApi)();
+
+  /** \brief Compute total size in bytes of the tensor data contained in an OrtValue.
+   *
+   * Returns the total number of bytes used to store the tensor data. For numeric tensors,
+   * this is sizeof(element_type) * total_element_count. For string tensors, this is the
+   * sizeof(std::string) * total_element_count. The latter does not take into account individual
+   * string buffers.
```

Comment:
is it meaningful to support string tensors in this API? E.g., APIs like `GetTensorMutableData` don't support string tensors. we also have `GetStringTensorDataLength()` for string tensors.

Reply:
If we do not support strings, should we return an error?